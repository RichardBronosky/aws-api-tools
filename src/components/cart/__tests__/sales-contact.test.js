import React from 'react'
import renderer from 'react-test-renderer'
import SalesContact from '../cart-parts/sales-contact'

describe('SalesContact', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SalesContact />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
