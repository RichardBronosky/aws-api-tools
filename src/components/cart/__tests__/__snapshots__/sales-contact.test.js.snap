// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SalesContact renders correctly 1`] = `
Array [
  <p
    className="sales-contact-text cell small-12"
  >
    Internet Sales Support Staff
  </p>,
  <p
    className="sales-contact-phone cell small-12"
  >
    <a
      href="tel:+18887095380"
    >
      1-888-709-5380
    </a>
  </p>,
  <p
    className="sales-contact-email cell small-12"
  >
    <a
      href="mailto:InternetSalesSupport@roomstogo.com"
    >
      internetSalesSupport@roomstogo.com
    </a>
  </p>,
  <p
    className="cell small-12 sales-contact-times"
  >
    Mon - Sat: 9AM - 10PM EST
  </p>,
  <p
    className="cell small-12 sales-contact-times"
  >
    Sunday: 11AM - 7PM EST
  </p>,
]
`;
